app/src/conferences/models.py

```diff
@@ -44,7 +44,8 @@ def validate_iban(value):
     m = re.match(iban_reg, value)
     if not m:
         raise ValidationError(_('This is not a valid Polish IBAN number'))
-    # https://pl.wikipedia.org/wiki/Mi%C4%99dzynarodowy_numer_rachunku_bankowego#Sprawdzanie_i_wyliczanie_cyfr_kontrolnych
+    # https://pl.wikipedia.org/wiki/Mi%C4%99dzynarodowy_numer_rachunku_bankowego
+    # #Sprawdzanie_i_wyliczanie_cyfr_kontrolnych
```

Comment:
URLs shouldn't be spited into multiple lines. If your formatter don't like this line, consider adding `# noqa` at the end.

Reply:
I turned off formatter for this line :)